src/main/scala/nl.knaw.dans.easy.properties/app/graphql/types/Scalars.scala

```diff
@@ -30,7 +30,7 @@ trait Scalars {
 
   private def parseUUID(s: String): Either[Violation, UUID] = {
     Either.catchNonFatal { UUID.fromString(s) }
-      .fold(_ => Left(UUIDCoercionViolation), Right(_))
+      .fold(_ => Left(UUIDCoercionViolation), _.asRight)
```

Comment:
Great find! In that sense you might also want to do 

```suggestion
      .fold(_ => UUIDCoercionViolation.asLeft, _.asRight)
```

Same below.